### ISSUE TITLE
Column Select with mouse no longer works on macOS High Sierra

### ISSUE THREAD

**Issue by JimCox-LogicDesignService** [Komodo/KomodoEdit#3621]
### Short Summary
Selecting a column of text using Alt(Option)+Left Mouse Button no longer works on macOS High Sierra.

### Steps to Reproduce
Open text file with multiple lines.  Hold down Alt(Option), click and hold down left mouse button, drag to select a column of text.  A column is selected as long as the left mouse button remains pressed.  After releasing the mouse button, only a single line stays selected.

When holding down Alt+Ctrl and dragging with the mouse, the column does stay selected after releasing the mouse button.  However, this also causes the pop-up contextual menu to appear (just like holding Ctrl and clicking left mouse button).

### Expected results
Column remains selected after releasing mouse button.

### Actual results
Only a single line remains selected after releasing the mouse button.

### Platform Information
Komodo Edit 11.1.0 Build 18196
iMac with macOS High Sierra 10.13.6 (plus all other software/security updates at this level)

### Additional Information
The column select operates normally when using only the keyboard.
The column select operates normally on my MacBook Pro with Mojave.

I reinstalled Komodo Edit on my iMac but the issue remained.  Could there be a local setting that is corrupt?  Messed up keybindings?

**Comment by cgchoffman** [comment]
Can you confirm this is broken with the default keybinding set?  Column select is working fine for me.

**Comment by JimCox-LogicDesignService** [comment]
Yes - I tried the Default set and the behavior is the same.  Let me know if there are any local settings/preference files I can temporarily move.

**Comment by cgchoffman** [comment]
This sounds like either an addon or a tool in your toolbox is causing the issue.  Couldn't hurt to share your logs as well when you reproduce the issue; Help > troubleshooting > View Logs

**Comment by cgchoffman** [comment]
Ya i'd agree with that assessment.  Thanks for doing that leg work.  I'm not sure why this would effect this issue but there are a LOT of errors regarding your selected colour scheme. You should change your colour scheme at the very least and try again.  If not, your best best is to refresh your prefs and start from scratch.  

Did you also upgrade Komodo when you update OSX or was the OS update the only change?

**Comment by JimCox-LogicDesignService** [comment]
I recently installed an SSD in my iMac and performed a clean install of High Sierra.  Komodo Edit was installed shortly after that.  There have been several small OS updates since then. Obviously, I made quite a few customizations to Komodo since it was installed. I’ll just start over with the default preferences since the column select feature is more important than the colors I had set up.

**Comment by JimCox-LogicDesignService** [comment]
After some additional experimentation, I discovered that this line in prefs.xml seems to cause the mouse-based column select issue:

`<boolean id="enableMultipleSelectionMouse">0</boolean>`

It's located in the global section of the prefs.xml file:

```
<preference-set id="global">
  <string id="analyticsLastVersion">11.1.0</string>
  <boolean id="analytics_enabled">1</boolean>
  <string id="analytics_ga_uid">############################</string>
  <string id="autoAbbreviationTriggerCharacters">\t</string>
  <boolean id="codeintelAutoInsertEndTag">1</boolean>
  <boolean id="codeintel_completion_auto_fillups_enabled">0</boolean>
  <string id="defaultEncoding">utf-8</string>
  <string id="defaultSelectedPref">editKeys</string>
  <string id="editFoldStyle">squaretree</string>
  <boolean id="editShowMinimap">0</boolean>
  <boolean id="editSmartSoftCharacters">1</boolean>
  <boolean id="editSmartWrapSelection">1</boolean>
  <long id="editWrapType">0</long>
  <string id="editor-font">Monaco</string>
  <string id="editor-scheme">Classic</string>
  <boolean id="enableAutoAbbreviations">1</boolean>
  <boolean id="enableMultipleSelectionMouse">0</boolean>
  <string id="filtered-prefs-open-nodes-by-id">{}</string>
  <boolean id="find-confirmReplacementsInFiles">1</boolean>
```

Removing that line or changing the property to `1` seems to fix the mouse-based column select issue.

`<boolean id="enableMultipleSelectionMouse">1</boolean>`

I was able to revert to my original prefs.xml by simply deleting that line.  The following "Smart Editing" Preference pane option appears to control that line in the prefs.xml:

"Ctrl+Click creates multiple selections"

![preferences](https://user-images.githubusercontent.com/5649755/48132225-6be57780-e258-11e8-964e-03eb1fb092ac.jpg)

I'm wondering if there is a macOS vs. Linux issue with this setting.  In Linux, Ctrl+Click does appear to add another edit point (blinking "I" cursor) where on macOS, Ctrl+Click brings up the contextual menu.  The right mouse button brings up the contextual (pop-up) menu in both macOS and Linux.

**Comment by cgchoffman** [comment]
I really appreciate your digging but I don't think that's the base issue.  That feature is enabled by default and no one else has reported an issues, granted it is early into Mojave's release.  I am not able to reproduce any issue with that feature enabled.  

But hey, at least it's working for you!

**Comment by JC-FPGA** [comment]
The mouse-based column select **works** when I have that option checked.  It stops working when that option is unchecked.  I wouldn't think that the "Ctrl+Click creates multiple selections" option is related to the column select, but it must be somehow.

**Comment by cgchoffman** [comment]
Classic.  Sorry, I swapped the bits in my head.  Thanks for clarifying, @JC-FPGA.
